flink-table/flink-table-code-splitter/src/test/java/org/apache/flink/table/codesplit/JavaCodeSplitterTest.java

```diff
@@ -20,28 +20,28 @@
 import org.apache.flink.core.testutils.FlinkMatchers;
 import org.apache.flink.util.FileUtils;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import java.io.File;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.HamcrestCondition.matching;
 
 /** Tests for {@link JavaCodeSplitter}. */
-public class JavaCodeSplitterTest {
+class JavaCodeSplitterTest {
 
     @Test
-    public void testSplitJavaCode() {
+    void testSplitJavaCode() {
         runTest("TestSplitJavaCode", 100, 3);
     }
 
     @Test
-    public void testNotSplitJavaCode() {
+    void testNotSplitJavaCode() {
         runTest("TestNotSplitJavaCode", 4000, 10000);
     }
 
     @Test
-    public void testInvalidJavaCode() {
+    void testInvalidJavaCode() {
         try {
             JavaCodeSplitter.split("public class InvalidClass { return 1; }", 4000, 10000);
```

Comment:
assertThatThrownBy

Reply:
(the current test is in fact incorrect since it still passes if no exception is thrown)

Reply:
Yes, thanks.
Agree. I noticed that the code throws nothing in case of syntax errors. I added this throwing.
And another point: it looks the code does not check anything if the code length is less or equal than the specified max method length... Currently left it as it is

Reply:
@tsreaper can you weigh in on this?

Reply:
@tsreaper ping

Reply:
@snuyanzin Can you revert the changes to the ReturnValueRewriter?

We'll rewrite `testInvalidJavaCode()` as originally planned, but disable it and file a ticket.